configure

```diff
@@ -92,7 +85,51 @@ def shellmessage(title,cmd,env=None,message=None,hidecmd=False,cwd=None,shell=Fa
         raise e
     return p
 
-message("Python",python_version)
+def get_requires_python_version(pyproject_path: Path = Path("pyproject.toml")) -> Tuple[int, int]:
```

Comment:
I get the need for this function but it's big, and is somewhat out of place in a configure script. If we really need to include it (and let's revisit that) I'd rather put it at the bottom than the top.